src/modules/copied/actions.js

```diff
@@ -0,0 +1,46 @@
+import e from 'gEngine/engine'
+import * as metricActions from 'gModules/metrics/actions'
+
+function isAtLocation(metric, location) {
+  return metric.location.row === location.row && metric.location.column === location.column
+}
+
+export function copy(spaceId){
+  return (dispatch, getState) => {
+    const state = getState()
+
+    const location = state.selection
+    const metric = Object.assign({}, state.metrics.find(m => m.space === spaceId && isAtLocation(m, location)))
+    const guesstimate = state.guesstimates.find(g => g.metric === metric.id)
+
+    dispatch({type: "COPY", copied: {metric, guesstimate}})
+  }
+}
+
+export function paste(spaceId){
+  return (dispatch, getState) => {
+    const state = getState()
+    if (!(state.copied && state.selection)) { return }
+
+    const {metric, guesstimate} = state.copied
+
+    const location = state.selection
+    const spaceMetrics = getState().metrics.filter(m => m.space === spaceId)
+    const existentMetric = spaceMetrics.find(m => isAtLocation(m, location))
+    const existingReadableIds = spaceMetrics.map(m => m.readableId)
+
+    let newMetric = e.metric.create(existingReadableIds)
+    newMetric.location = location
+    const newItem = Object.assign({}, metric, newMetric)
```

Comment:
That doesn't appear to work. Perhaps I'm misunderstanding, but as written that doesn't appear to work.